components/src/core/EmptyState/EmptyState.tsx

```diff
@@ -26,55 +33,72 @@ export type EmptyStateProps = HTMLAttributes<HTMLDivElement> & {
     title: ReactNode;
 };
 
-const useStyles = makeStyles((theme: Theme) => ({
-    root: {
-        color: theme.palette.text.primary,
-        height: '100%',
-        minHeight: '100%',
-        display: 'flex',
-        justifyContent: 'center',
-        flexDirection: 'column',
-        textAlign: 'center',
-        alignItems: 'center',
-        padding: '1rem',
-    },
-    icon: {
-        color: theme.palette.text.secondary,
-        marginBottom: '1rem',
-        display: 'flex',
-        fontSize: 96,
-    },
-    description: {
-        color: theme.palette.mode === 'dark' ? theme.palette.text.secondary : theme.palette.text.primary,
-    },
-    actions: {
-        marginTop: '1rem',
-    },
+const Root = styled(Box, {
+    name: 'empty-state',
+    slot: 'root',
+})(({ theme }) => ({
+    color: theme.palette.text.primary,
+    height: '100%',
+    minHeight: '100%',
+    display: 'flex',
+    justifyContent: 'center',
+    flexDirection: 'column',
+    textAlign: 'center',
+    alignItems: 'center',
+    padding: '1rem',
+}));
+
+const Icon = styled(Box, {
+    name: 'empty-state',
+    slot: 'icon',
+})(({ theme }) => ({
+    color: theme.palette.text.secondary,
+    marginBottom: '1rem',
+    display: 'flex',
+    fontSize: 96,
+}));
+
+const Description = styled(Typography, {
+    name: 'empty-state',
+    slot: 'description',
+})(({ theme }) => ({
+    color: theme.palette.mode === 'dark' ? theme.palette.text.secondary : theme.palette.text.primary,
+}));
+
+const Actions = styled(Box, {
+    name: 'empty-state',
+    slot: 'actions',
+})(() => ({
+    marginTop: '1rem',
 }));
 
 const EmptyStateRender: React.ForwardRefRenderFunction<unknown, EmptyStateProps> = (
     props: EmptyStateProps,
     ref: any
 ) => {
-    const { actions, classes, description, icon, title, ...otherDivProps } = props;
-    const defaultClasses = useStyles(useTheme());
+    const { actions, classes, description, icon, title, ...otherProps } = props;
+    const ownerState = {
+        ...props,
+    };
+    const defaultClasses = useUtilityClasses(ownerState);
+
```

Comment:
```suggestion
    const defaultClasses = useUtilityClasses(props);
```
we can pass props directly instead of storing it in this ownerState object